setup.py

```diff
@@ -1,4 +1,11 @@
 #!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This file is part of Viper - https://github.com/viper-framework/viper
+# See the file 'LICENSE' for copying permission.
+from __future__ import unicode_literals
+from __future__ import absolute_import
```

Comment:
I worked on a project last year where I invested some time in checking these __future__ imports and from which I copied the code without re-checking. In that project I had to make sure it worked on Python 2.6 as I had a machine running RHEL 6. 

Sorry for not checking.. I'll look into this again and clean that stuff out.